pkg/sql/doctor/doctor.go

```diff
@@ -339,18 +287,13 @@ func ExamineJobs(
 }
 
 func reportMsg(desc catalog.Descriptor, format string, args ...interface{}) string {
-	var header string
-	switch desc.(type) {
-	case catalog.TypeDescriptor:
-		header = "    Type"
-	case catalog.TableDescriptor:
-		header = "   Table"
-	case catalog.SchemaDescriptor:
-		header = "  Schema"
-	case catalog.DatabaseDescriptor:
-		header = "Database"
+	msg := fmt.Sprintf(format, args...)
+	// Add descriptor-identifying prefix if it isn't there already.
+	// The prefix has the same format as the validation error wrapper.
+	msgPrefix := fmt.Sprintf("%s %q (%d): ", desc.TypeName(), desc.GetName(), desc.GetID())
+	if msg[:len(msgPrefix)] == msgPrefix {
+		msgPrefix = ""
```

Comment:
Note: this test is required if we want to make the namespace table report messages share the same format as the descriptor validation error messages.